Update lint-format.md (#4245)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/dbt-cloud-ide/lint-format.md

- Works with Jinja and SQL, 
- Comes with built-in [linting rules](https://docs.sqlfluff.com/en/stable/rules.html). You can also [customize](#customize-linting) your own linting rules.
- Empowers you to [enable linting](#enable-linting) with options like **Lint** (displays linting errors and recommends actions) or **Fix** (auto-fixes errors in the IDE).
- Displays a **Code Quality** tab to view code errors, and provides code quality visibility and management.

:::Ephemeral models not supported

> Linting doesn't support ephemeral models in dbt versions 1.5 and lower. Refer to the [FAQs](#faqs) for more info.

Reviewer: ```suggestion
Linting doesn't support ephemeral models in dbt v1.5 and lower. Refer to the [FAQs](#faqs) for more info.
```

Make sure you're on a development branch. Formatting or Linting isn't available on "main" or "read-only" branches. 
</details>

<details>

> <summary>Why is the lint operation inconsistent with running SQLFluff out of the dbt Cloud IDE (e.g. in a GitHub Action)?</summary>

Reviewer: ```suggestion
<summary>Why is there inconsistent SQLFluff behavior when running outside the dbt Cloud IDE (such as a GitHub Action)?</summary>
```

> * Double check that your SQLFluff version that is running outside of the dbt Cloud IDE is consistent with the version used within the dbt Cloud IDE (which is usually printed out in the "Code Quality" tab after a lint operation).

Reviewer: ```suggestion
* Double-check your SQLFluff version matches the one in dbt Cloud IDE (found in the "Code Quality" tab after a lint operation).
```

> * If your lint operation is always passing when there exist clear rule violations - check that you are not linting a model that uses ephemeral models (dbt versions prior to 1.6 only).

Reviewer: ```suggestion
* If your lint operation passes despite clear rule violations, confirm you're not linting models with ephemeral models. Linting doesn't support ephemeral models in dbt v1.5 and lower. 
```
